src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -112,5 +112,8 @@ public static class EnvironmentSettingNames
 
         // Machine identifier
         public const string AntaresComputerName = "COMPUTERNAME";
+
+        public const string AppKind = "APP_KIND";
+        public const string WorkFlowAppKind = "workflowapp";
```

Comment:
It looks like this static class is meant to contain environment variable names and not values.  Is there a more appropriate place to specify this string constant?

Reply:
yeah my bad. There is a constant class. Will move it there.